nix/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
I don't think it's useful because adding anything here would require modifying the root `flake.nix` as well to import it, and it is under version control.

Moreover, this `.gitignore` can actually prevent having any local changes that will not be pushed to the repo, because the most useful strategy to do so is to have private commits on local main and use something like https://github.com/keith/git-pile to manage patches on top. I actually did this when I had some private changes to `shell.nix` that weren't ready to be shared with the rest of the team.

Reply:
Ok, I can remove it.